src/momento/aio/_scs_data_client.py

```diff
@@ -234,5 +233,24 @@ async def multi_get(
             _momento_logger.debug(f"multi_get failed with response: {e}")
             raise _cache_service_errors_converter.convert(e)
 
+    async def delete(
+        self, cache_name: str, key: Union[str, bytes]
+    ) -> cache_sdk_ops.CacheDeleteResponse:
+        _validate_cache_name(cache_name)
+        try:
+            _momento_logger.debug(f"Issuing a delete request with key {str(key)}")
```

Comment:
just as im looking at this noticed we could probably DRY up our access logging. Maybe for follow up pr but we could probably extract this debug logging out into some standard decorator vs writing out in every method.

Reply:
this is true.


Also, this strategy for logging is not great. You're (1) encoding, (2) copying the key and (3) allocating yet a 3rd copy of the memory just to throw it away probably 99.99999% of the time or more. Why does this need to do all of that work when the `debug` level is not enabled?

Reply:
oh geeze, here & everywhere i see.

Reply:
Made issue to revisit. Agree @kvcache

Reply:
you can consider declining to propagate this bad pattern by merging [this fix](https://github.com/momentohq/client-sdk-python/pull/102) and updating your net new logging statements

Reply:
@malandis Did you miss this thread?